src/blueprint.zig

```diff
@@ -270,3 +308,5 @@ pub const Blueprint = struct {
 		}
 	}
 };
+
+const sbb = main.server.terrain.structure_building_blocks;
```

Comment:
Why is this at the bottom of the file?

Reply:
I think I was searching for source of stack overflow in refAllDeclsRecursiveExceptCImports, tried if order of definitions matters and didn't clean up afterwards.